packages/@o3r/rules-engine/src/engine/fact/current-time/current-time.facts.ts

```diff
@@ -0,0 +1,12 @@
+import type { FactDefinitions } from '../index';
+
+
+/**
+ * Operator facts that provide the current time
+ */
+export interface CurrentTimeFacts extends FactDefinitions {
```

Comment:
If the fact is defined in this Engine and required by a default Operators, it needs to be registered by the Engine itself.
Up to the User to override this fact with its own impl to change the frequency

Reply:
got it